### PR TITLE
Angel Aliseda Alonso Homework 1

### PR DESCRIPTION
**Pull request by a-aliseda**
Mastering Markdown was a useful guide on how to use lists, bolds, italics and how to add links to edit the README file.

Challenges:
1.- It does not explain clearly how to embed images clearly so I had to look for it in StackOverflow and GitHub Communities
2.- I have my computer synced with Google Docs so it gets hard to work because an error message keeps popping up. I had to stop the automatic syncing to keep that message from popping up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
-# hw01
\ No newline at end of file
+# Computing for the Social Sciences: Homework 1
+## Angel Aliseda Alonso 
+
+### Background
```
Comment (nikhilmandala) [inline README.md:4]: Good use of different levels of headers

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+### Education
+
+* B.A. in Economics from the Instituto Tecnologico Autonomo de Mexico (ITAM), _2009-2014_
+* B.A. in Political Science from the Instituto Tecnologico Autonomo de Mexico (ITAM), _2009-2014_
+
+### Work Experience
+
+* __Mexican Social Security Institute (IMSS)__
+  * _Advisor to the Director of Strategic Planning, 2016-2018_
```
Comment (nikhilmandala) [inline README.md:15]: Nice use of multiple levels in a bullet list. I was trying to figure out how to do the same thing in my own assignment -- I learned something new!

File: README.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+    * Collaborated in designing the impact evaluation of a pay-for performance pilot program to       enhance metabolic control of diabetic patients using a randomized controlled trial by          developing some evaluation instruments and analyzing the baseline data used for the            randomization process
+    
+* __Instituto Tecnológico Autónomo De México (ITAM)__
+  * _Research Assistant at the Department of Political Science, 2013-2014_
+    * Collaborated in the impact evaluation of “Jóvenes con Porvenir,” a training program            targeted to unemployed youth of municipalities of the Mexican state of Jalisco, by             compiling the dataset and performing the initial statistical analysis
+    
+You can find more information about me in [LinkedIn](https://www.linkedin.com/in/angel-aliseda-alonso-06a7aa74/)
+
+### Pictures
+![Picture of Angel](AAA.jpg)
```
Comment (nikhilmandala) [inline README.md:29]: Your image doesn't seem to render properly on GitHub. I had the same problem in my own homework assignment -- I think you need to specify the file path more specifically? This is something I need to check with the TAs about

File: README.md [hunk 4]
```diff
@@ -1 +1,33 @@
...
+    * Coordinated a 12-person team responsible for implementing a bed management program and a       new post-triage process in emergency departments aimed to increase staff coordination and       reduce waiting time at 180 general hospitals
+  
+  * _Analyst at the Strategic Research Area, 2014-2016_
+    * Collaborated in designing the impact evaluation of a pay-for performance pilot program to       enhance metabolic control of diabetic patients using a randomized controlled trial by          developing some evaluation instruments and analyzing the baseline data used for the            randomization process
+    
+* __Instituto Tecnológico Autónomo De México (ITAM)__
+  * _Research Assistant at the Department of Political Science, 2013-2014_
+    * Collaborated in the impact evaluation of “Jóvenes con Porvenir,” a training program            targeted to unemployed youth of municipalities of the Mexican state of Jalisco, by             compiling the dataset and performing the initial statistical analysis
+    
+You can find more information about me in [LinkedIn](https://www.linkedin.com/in/angel-aliseda-alonso-06a7aa74/)
```
Comment (nikhilmandala) [inline README.md:26]: The link is properly embedded in the text

File: README.md [hunk 5]
```diff
@@ -1 +1,33 @@
...
+
+* __Mexican Social Security Institute (IMSS)__
+  * _Advisor to the Director of Strategic Planning, 2016-2018_
+    * Participated in the impact evaluation of a bed management program at general hospitals by       developing the main impact indicators for the program, composing the dataset from IMSS         hospital’s administrative data, performing the econometric analysis and helping to             elaborate the final impact report
+    * Coordinated a 12-person team responsible for implementing a bed management program and a       new post-triage process in emergency departments aimed to increase staff coordination and       reduce waiting time at 180 general hospitals
+  
+  * _Analyst at the Strategic Research Area, 2014-2016_
+    * Collaborated in designing the impact evaluation of a pay-for performance pilot program to       enhance metabolic control of diabetic patients using a randomized controlled trial by          developing some evaluation instruments and analyzing the baseline data used for the            randomization process
+    
+* __Instituto Tecnológico Autónomo De México (ITAM)__
```
Comment (nikhilmandala) [inline README.md:22]: Great use of bold, italics, and bullets

**Review by nikhilmandala — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |           |              |   X         |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Elaborate on above, especially for "needs work."

Your image didn't render properly on GitHub. Next time it may be helpful to preview your code before submitting, just to catch minor errors like this!

* Some specific praise?

Overall, this was a well-organized and informative bio!

* Something I learned?

How to display a bulleted list with multiple levels of indents

* Specific constructive criticism?

Nothing specifically, except for maybe just previewing your code before submitting. Everything was great except for the image!

* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.

I'm not sure exactly how to fix the image issue yet, but I think it's related to the file path of the image, and the fact that we're uploading it online... I would check with the TA's though.

**Review by victorhugovazquez — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Excellent, Angel! 

Remarks:

* Elaborate on above, especially for "needs work."
* Some specific praise?
He mentioned in detail about his background in a professional way, using bullets and bold style. 
* Something I learned?
I had a problem with my images, so looking at this work it helped me to learn how to do it. 

He covered the ground of the assignment well, with lists and headers and italics and bold. Very nice job!

File: README.md (hunk 3 above)
Comment (victorhugovazquez) [reply README.md:29]: Hi, I cloned it into my desktop and the image rendered well.

Comment (bensoltoff) [reply README.md:29]: Or if you view it directly [on the website it works](https://github.com/cfss-fa19/hw01/blob/51c98e8fe708e7d492a1864c034c2a0b0474846c/README.md). How are you trying to view the file @nikhilmandala?

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |        x   |              |            |

* Regarding the Google Drive error. Yes, that happens very frequently if you store a Git repository in a Google Drive folder. In fact, I recommend not doing that and storing it in a separate location on your computer. The nice thing is that when you push your commits to GitHub, you are backing up your repository as well so you don't really need to use Google Drive for those files

**Pull request closed without merging**